<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardProgress</class>
 <widget class="QWidget" name="WizardProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>274</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QLabel" name="wizardTittleLabel">
     <property name="text">
      <string>Wizard Progress:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>117</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/step1_red.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/division_line_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Networks</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblChkImage1">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/green_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/step2_red.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/division_line_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Phones</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblChkImage2">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/green_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/step3_red.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/division_line_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Audio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblChkImage3">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/green_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/step4_red.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/division_line_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Video Phone</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblChkImage4">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/green_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/step5_red.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_14">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>3</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/division_line_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblChkImage5">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>21</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/wengophone.qrc">:/pics/wizard/green_checkmark.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
 </resources>
 <connections/>
</ui>
